spec/presenters/planned_disbursement_xml_presenter_spec.rb

```diff
@@ -0,0 +1,25 @@
+require "rails_helper"
+
+RSpec.describe PlannedDisbursementXmlPresenter do
+  let(:planned_disbursement) { build_stubbed(:planned_disbursement) }
+
+  describe "#period_start_date" do
+    it "returns a human readable date" do
+      planned_disbursement.period_start_date = "2020-06-25"
```

Comment:
These tests are simply returning the date as it was inputted, which is fine, but it would be useful to have a test to show it actually reformatting a date, e.g. 25-06-2020 -> 2020-06-25